Xamarin.Forms.Platform.WPF/Extensions/ImageExtensions.cs

```diff
@@ -26,12 +26,7 @@ public static Stretch ToStretch(this Aspect aspect)
 			}
 		}
 
-		public static WImageSource ToWindowsImageSource(this ImageSource source)
-		{
-			return source.ToWindowsImageSourceAsync().GetAwaiter().GetResult();
-		}
-
-		public static async Task<WImageSource> ToWindowsImageSourceAsync(this ImageSource source, CancellationToken cancellationToken = default(CancellationToken))
+		public static async Task<WImageSource> ToWindowsImageSourceAsync(this ImageSource source, CancellationToken cancellationToken = default)
```

Comment:
the WPF project still use `<LangVersion>default</LangVersion> instead of 7.3 as it overrides the Directory.Build.props. Either remove the LangVersion from the WPF csproj, or do not use default literal.

Reply:
Okay, good to know.